src/plugin/cesium/tiles/cesium3dtileslayer.js

```diff
@@ -65,10 +77,24 @@ plugin.cesium.tiles.Layer.prototype.synchronize = function() {
 
   if (!this.hasError()) {
     var tilesetUrl = '';
-    if (!isNaN(this.assetId)) {
+    if (!isNaN(this.assetId) && this.accessToken) {
       tilesetUrl = Cesium.IonResource.fromAssetId(this.assetId, {
         accessToken: this.accessToken
       });
+
+      tilesetUrl.then(() => {
+        // We don't care to note that it resolves, we just want a response if it doesn't
+      }, () => {
+        // Clear the saved access token because it was rejected
+        const layerId = this.getId();
+        os.settings.set(`plugin.cesium.SettingsKey.ACCESS_TOKEN.${layerId}`, '');
+
+        var errorMsg = 'The provided access token was rejected. ' +
+        'Turn the layer off and back on to provide another access token.';
+        this.setTokenError_(errorMsg);
+        this.updateError();
+        this.getIcons();
```

Comment:
No need to call `this.getIcons();` after the error is updated. When the error changes, the layer node will respond to the dispatched event and update the icons.